### ISSUE TITLE
Loginized under Gnome 3.36.

### ISSUE THREAD

**Issue by Tuxman2** [juhaku/loginized#27]
Hi,

I appreciate your software, and it does a good job. But now you say that it is obsolete under Gnome 3.36 and that is bad news. So what's the solution now to replace Loginized ? I guess I will have to manually edit files? Is there another solution, and which one?

Thank you for your help.

**Comment by juhaku** [comment]
Hello, 
Just tested it with Manjaro 19.0.2 with all updates from upstream having gnome-shell version 3.36. It seemed to work well at least changing the background image. The disclaimer is there just to notify that there might be some unexpected behaviour with the never version of gnome.

**Comment by thiggy01** [comment]
It's not working with the new Ubuntu 20.04 Focal Fossa.

**Comment by Tuxman2** [comment]
Hello,

> Hello,
> Just tested it with Manjaro 19.0.2 with all updates from upstream having gnome-shell version 3.36. It seemed to work well at least changing the background image. The disclaimer is there just to notify that there might be some unexpected behaviour with the never version of gnome.

I also use Manjaro 19.0.2 with all the new updates (Gnome 3.36 included). Do you think if I change the background image and another theme only under your last version (1.4.0), I can encounter problems with Gnome 3.36 ?

**Comment by thiggy01** [comment]
I just tested with Ubuntu 20.04 and even compiled Yaru as you sad in your README, but it messed up my Yaru settings and I couldn't enter a gnome-session anymore.
I had to reinstall yaru-theme package from Ubuntu repositories in order to be able to enter gnome again!

**Comment by juhaku** [comment]
@Tuxman2 Yeah but please note that there might occur issues when theme changing is used. See the topic at this issue: https://github.com/juhaku/loginized/issues/19. That is not a problem of the application it's the problem of the themes that are not compatible with the never version of gnome-shell. Moreover the breaking change in gnome-shell was introduced in version 3.34.

@thiggy01 That's sad to hear. Ubuntu always does things little differently. Okay. the Yaru theme did work with Ubuntu 19.10 version. I guess I need to update the documentation a bit. Good if you got things sorted out, but bad that it did not work after all. Not sure whether I am going to troubleshoot the issues with it further as I am not using Gnome desktop anymore for quite some time already.

**Comment by Tuxman2** [comment]
Hello,

> @Tuxman2 Yeah but please note that there might occur issues when theme changing is used. See the topic at this issue: #19. That is not a problem of the application it's the problem of the themes that are not compatible with the never version of gnome-shell. Moreover the breaking change in gnome-shell was introduced in version 3.34.

I know, I tried some themes with Loginized and I encounter problems with them under Gnome 3.36. The default option in the Loginized menu works, but it give me the cyan color from Manjaro that I don't like on the Login screen. I removed the manjaro-gdm-theme, manjaro-gnome-assets packages to have the default color from Gnome (blue), but each time I change the background through Loginized, the default Manjaro cyan color come back (Theme is set to Default in Loginized).

**Comment by juhaku** [comment]
> Hello,
> 
> > @Tuxman2 Yeah but please note that there might occur issues when theme changing is used. See the topic at this issue: #19. That is not a problem of the application it's the problem of the themes that are not compatible with the never version of gnome-shell. Moreover the breaking change in gnome-shell was introduced in version 3.34.
> 
> I know, I tried some themes with Loginized and I encounter problems with them under Gnome 3.36. The default option in the Loginized menu works, but it give me the cyan color from Manjaro that I don't like on the Login screen. I removed the manjaro-gdm-theme, manjaro-gnome-assets packages to have the default color from Gnome (blue), but each time I change the background through Loginized, the default Manjaro cyan color come back (Theme is set to Default in Loginized).

When you delete the manjaro-gdm-theme, after that you need to install the default gdm them from somewhere. The thing is that after first use of Loginized app the app has cached the default theme at that point. I suppose it is the manajaro-gdm-theme. After you have installed the default blue theme. You can open the app -> select settings (weel icon) -> and click or drag and drop the new global theme to the big box. I after installing the blue theme it will install it to __/usr/share/gnome-shell__ folder.. there should be file called gnome-shell.theme.gresource that is just recently modified. You may use that file as new default when you update it via application. Another way to do this is actually delete folder from __.config/Loginized/default__. May be that the loginized is lowercase. That folder contains the cached default theme from first use. And will be populated if not found.

Hope this helps.

**Comment by Tuxman2** [comment]
@juhaku: Thanks for your help. I don't like that ugly cyan color. LOL.
I will try your tip and will tell you if it worked. ;-)

Regards.

News from April 1th: @juhaku: Your tip works. :-)

I have one question. I extracted the file gnome-shell-theme.gresource by default from GDM and I modified some elements from it. But when I changed the high contrast option in the screen, the background I defined vanished to offer a grey color. I tried to modify the gnome-shell-high-contrast.css file to define a background image in it, but it caused problems (bugs on the display of the screen and gdm menus). It is not possible to define a background in the gnome-shell-high-contrast.css file like in the gnome-shell.css file ?

Regards.

**Comment by juhaku** [comment]
@Tuxman2 

> Your tip works. :-)

Great.

>  ...It is not possible to define a background in the gnome-shell-high-contrast.css file like in the gnome-shell.css file ?

That actually I don't know, I never tried to use the high contranst css file. But in theory they should act in similar fashion. So it should work. But might be that the high contranst css does not support that background image modification. or the structure is so different that it just does not work as global gnome shell theme. You can see if the high-contrast.css file contains something similar to this: 
```
#lockDialogGroup { 
   background: #2e3436 url(\"resource:\/\/\/org\/gnome\/shell\/theme\/texture.png\");
}"
```
If it does then it suppose to support it as well. But I might be wrong But using it as base for system theme might also be the show stopper. This could only be verified by someone from Gnome development team I suppose.

**Comment by reverland** [comment]
> I just tested with Ubuntu 20.04 and even compiled Yaru as you sad in your README, but it messed up my Yaru settings and I couldn't enter a gnome-session anymore.
> I had to reinstall yaru-theme package in order to reset its settings and to be able to enter gnome again!

Ubuntu patched gnomeshell default gdm mode to use gdm3.css or gdm3-theme.gresource for gdm session mode lonely. You may add a mode file in modes directory for gdm to overide this behavior.
